docs/coverage.md

```diff
@@ -1,12 +1,12 @@
-#### [Safe Apps liveness](./../src/safe-apps/working_apps.test.js)
+#### [Safe Apps liveness](./../src\safe-apps/working_apps.test.js)
```

Comment:
@DiogoSoaress could you help Franco set up mincgw or WSL to avoid these slashes?

Reply:
Yes, we can look at that.

Reply:
Maybe we should just replace backslashes in the script that generates this file. I'll make a ticket for that.

Reply:
Yes, editing the script will do the trick. But @francovenica is installing a linux terminal on his editor anyway for the time being.